application/models/User_Consumption_LoginRole.php

```diff
@@ -30,8 +33,9 @@ public function get($roleId) {
 
         foreach ($users as $key => $user) {
             $users[$key][Consumption::FIELD_AMOUNT] = $this->Consumption->get($user[Login::FIELD_LOGIN_ID]);
+            $users[$key][Transaction::FIELD_TIME] = $this->User_Transaction->getLatestForSubjectId($user[$this->Login::FIELD_LOGIN_ID]);
```

Comment:
Can't `$this->Login::FIELD_LOGIN_ID` just be `Login::FIELD_LOGIN_ID`? At least that's how it's done the line above, so if possible it would be good to keep it consistent.